lib/model/message_list.dart

```diff
@@ -381,6 +381,7 @@ class MessageListView with ChangeNotifier, _MessageSequence {
     for (final message in result.messages) {
       if (_messageVisible(message)) {
         _addMessage(message);
+        store.recentSenders.handleMessage(message);
```

Comment:
The ideal version of this would go on top of a central message store, #648, so that `MessageListView` wouldn't have to know individually about all the miscellaneous data structures that care about a summary of messages.

This PR doesn't need to block on that one; for now, it's fine to add these calls in a more ad-hoc way like this.

The current version of this PR doesn't call this `RecentSenders.handleMessage` method in enough cases, though. Compare which messages get passed to `reconcileMessages` in #648.